src/ui/public/config/defaults.js

```diff
@@ -20,6 +23,12 @@ define(function () {
         value: 'MMMM Do YYYY, HH:mm:ss.SSS',
         description: 'When displaying a pretty formatted date, use this format',
       },
+      'dateFormat:tz': {
+        value: 'Default',
```

Comment:
I'd use either "browser" or "local" to try to make it clear how the detection is being done. Old versions of kibana used "browser"